Snippets/Snippets_4/DelayedDelivery/DeferForTimeSpan.cs

```diff
@@ -8,6 +8,9 @@ class DeferForTimeSpan
         public void SendDelayedMessage()
         {
             IBus bus = null;
+            #region configure-timeout-manager
+            // Timeout Manager is part of NServiceBus Core, no need to call anything.
```

Comment:
not sure i like snippets with just a comment. perhaps this should be captured in the text somehow?